frontend/components/feedback-task/container/fields/RecordFieldsHeader.vue

```diff
@@ -47,21 +47,11 @@ export default {
       type: Array,
     },
   },
-  data() {
-    return {
-      animateCheckboxes: false,
-    };
-  },
   methods: {
     onSelectedRecord(isSelected) {
       this.$emit("on-select-record", isSelected, this.record);
     },
   },
-  created() {
-    this.$nuxt.$on("animate-checkboxes", () => {
-      this.animateCheckboxes = true;
-    });
-  },
```

Comment:
If we remove the events, the animation will be shown every time the record is rendered and we only want it to be shown when the bulk component is rendered